hub/gads-ui/src/components/DeviceControl/Tabs/Actions/Apps/UninstallApp.js

```diff
@@ -30,7 +31,9 @@ export default function UninstallApp({ udid, installedApps }) {
         }
 
         api.post(url, body)
-            .then(() => {
+            .then((response) => {
+                setInstalledAppsList(response.data.result)
```

Comment:
Add validation to ensure `response.data.result` is defined and properly formatted before updating `setInstalledAppsList`

Reply:
We are not doing any runtime parsing with JSON across the whole UI and we are not using TS to assign the object to an interface type so I don't see a point to do this. For now it might be best to make sure the backend behaves as expected. I added a minimal safety check for the result object at least [here](https://github.com/shamanec/GADS/pull/185/commits/4957832b1908943c42adb380c3ebcb9b9852e2b1)